content/docs/code-splitting.md

```diff
@@ -247,21 +213,21 @@ const App = () => (
 
 ## Named Exports {#named-exports}
 
-`React.lazy` currently only supports default exports. If the module you want to import uses named exports, you can create an intermediate module that reexports it as the default. This ensures that tree shaking keeps working and that you don't pull in unused components.
+`React.lazy` sementara ini hanya dapat digunakan untuk modul dengan ekspor `default`. Jika modul yang ingin Anda impor menggunakan *named exports* (ekspor modul yang diberi nama dan tidak menggunakan *default*), Anda dapat membuat modul perantara yang mengekspor ulang modul tersebut sebagai *default*. Cara ini memastikan proses *treeshaking* tetap berjalan dan tidak mengikutsertakan komponen yang tidak terpakai.
 
 ```js
-// ManyComponents.js
+// KumpulanKomponen.js
 export const MyComponent = /* ... */;
-export const MyUnusedComponent = /* ... */;
+export const KomponenTidakTerpakai = /* ... */;
```

Comment:
```suggestion
export const MyUnusedComponent = /* ... */;
```

Please don't translate file + variable names, as mentioned in the [translation guidelines](https://github.com/reactjs/id.reactjs.org/blob/master/TRANSLATION.md).